cypress/commons/actions/generic/ScenarioParameters.js

```diff
@@ -62,7 +62,7 @@ function collapseParametersAccordion() {
       }
     });
 }
-function edit(timeout = 5) {
+function edit(timeout) {
```

Comment:
Can we leave a default value in this function when we want to edit without waiting the end of a scenario run, or does it make the tests break ?

Reply:
In fact, a default value is already implemented at the lower level directly in the getter function :
`function getParametersEditButton(timeout = 4) {
  return cy.get(GENERIC_SELECTORS.scenario.parameters.editButton, { timeout: timeout * 1000 });
}`

Reply:
Even though a default value is set in `getParametersEditButton`, if we remove the default value in the `edit` function, **we can no longer call `edit` without parameter in the tests files**.
That means that even if we want to edit parameters without waiting the end of a scenario run, we have to set explicitly a waiting time -> some tests file could be simpler and easier to read without the timeout value